README.md

```diff
@@ -1,50 +1,65 @@
-3. Run `git remote add upstream https://github.com/oracle-devrel/devo.tutorials.git`
+
+```git clone https://github.com/<myprofile>/devo.tutorials.git```
+
+3. Add the remote upstream:
+
+```git remote add upstream https://github.com/oracle-devrel/devo.tutorials.git```
 
 ### Add/modify your content in a branch
 
-1. Run `git checkout main`  to make sure the new branch comes from master
-2. Create the new branch and switch to it: `git checkout -b <BRANCH_NAME>`
-3. Do your implementation, then _stage_ and _commit_ it
-   * `git add .`
-   * `git commit -am "insert your comments here"`
+1. Run `git checkout main`  to make sure the new branch comes from the main branch
+
+2. Create the new branch and switch to it: 
+
+```git checkout -b issue-<number>```
+
+3. Write your content, then _stage_ and _commit_ it
+   * ```git add .```
+   * ```git commit -s```
+
+4. Add your comments and use [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) to link to your issue.
+
+5. Push your changes to your fork:
+
+```git push origin issue-<number>``
 
 ### Submit a Pull Request (PR)
 
-1. Before submit a PR, check if any commits have been made to the upstream main by running:
-    * `git fetch upstream`
-    * `git checkout main`
-    * `git merge upstream/main`
-2. If any new commits, rebase your branch
-    * `git checkout <BRANCH_NAME>`
-    * `git rebase main`
-3. Commit and Push changes
-    * `git add .`
-    * `git commit -m "insert here your comments"`
-    * `git push origin <BRANCH_NAME>`
-4. Create a Pull Request for your branch on GitHub by visiting your forked repo page.
->When you create the PR, make sure to check the <ins>"***Allow edits by maintainers***"</ins> option in the PR. This allows for easy, rapid modifications that our tech editors might provide.
-5. After you submit the Pull Request, sit tight and look for any comments, edits, approvals or rejections!
+1. Create a Pull Request for your branch on GitHub by visiting your forked repo page.
+
+> When you create the PR, make sure to check the <ins>"***Allow edits by maintainers***"</ins> option in the PR. This allows for easy, rapid modifications that our tech editors might provide.
+
+2. After you submit the Pull Request, sit tight and look for any comments, edits, approvals or rejections!
 
 ### Cleanup / Align to the source repo
 
-1. Once your PR has been approved and merged, it's time to do some cleanup.
-2. Run `git pull upstream main`
-3. Remove merged branch
-    1. Run `git checkout main`  
-    2. Run `git branch -d <BRANCH_NAME>`
-4. Commit
-5. Update the master branch in your forked repo
-    1. Run `git push origin main`
-6. Remove the branch from your forked repo
-    1. Run `git push --delete origin <BRANCH_NAME>`
+1. Once your PR has been approved and merged, it's time to do update your local repo.
+
+2. Ensure you are on main:
+   ```git checkout main```
```

Comment:
one-line code spans use a single backtick, please. A newline must exist between a parent list item and indented paragraph/code content